scrape_data/combine_daily_files.py

```diff
@@ -20,15 +21,15 @@
 logging.basicConfig(level=logging.INFO)
 
 
-def combine_daily_files(date: str):
+def combine_daily_files(date: str, bucket_list: List[str]):
     """Combine raw JSON files returned by API into daily CSVs. 
 
     Args:
         date: Date string for which raw JSON files should be combined into CSVs. Format: YYYY-MM-DD.
     """
     s3 = boto3.resource("s3")
 
-    for bucket_name in [BUCKET_PRIVATE, BUCKET_PUBLIC]:
+    for bucket_name in bucket_list:
         logging.info(f"processing data from {bucket_name}")
         bucket = s3.Bucket(bucket_name)
         objects = bucket.objects.filter(Prefix=f"bus_data/{date}")
```

Comment:
This makes sense but I didn't want to add another dependency to this because it's actually deployed to the Lambda function and adding packages is kind of annoying for that, if that's ok?